corehq/apps/analytics/tasks.py

```diff
@@ -351,8 +352,10 @@ def send_hubspot_form_task(form_id, web_user, hubspot_cookie, meta,
 
 
 @analytics_task()
-def send_hubspot_form_task_v2(form_id, web_user, hubspot_cookie, meta,
+def send_hubspot_form_task_v2(form_id, web_user_id, hubspot_cookie, meta,
```

Comment:
How does it handle argument names changing? e.g. before it was web_user=x, now it's web_user_id=x

Reply:
I'm assuming that celery doesn't care about the names of positional arguments.  I believe ```args``` are stored as a tuple in the celery backend.  (If I'm wrong, worst case is we drop a small number of analytics events.)